content/source/docs/enterprise/getting-started/sentinel.html.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: "enterprise2"
+page_title: "Configuring Sentinel Policies - Getting Started - Terraform Enterprise"
```

Comment:
Without going too deep on this right now: we should try and get this material into learn.hashicorp.com, @tr0njavolta's bandwidth permitting. The getting-started stuff in the TFE docs could benefit from a pare-back; I'd like to reduce it to more of an overview of the tasks you need to do to get up and running, linking out to both learn and the per-feature docs for all of the heavy lifting. 

But in the meantime, something like this is a promising temporary measure. I think we'll have a better time if we preserve the existing `policies.html.md` filename instead of renaming — we can change the title as needed, but moving things means permanent redirects.